h/templates/accounts/login_oauth.html.jinja2

```diff
@@ -2,13 +2,22 @@
 
 {% block page_title %}Log in{% endblock %}
 
+{% block styles %}
+  {{ super() }}
+
+  {% for url in asset_urls('forms_css') %}
+    <link rel="stylesheet" href={{ url }} as="style"/>
```

Comment:
```suggestion
    <link rel="stylesheet" href="{{ url }}" >
```

3 changes: Make sure the URL is quoted, `as` is [not valid](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/link#as) for `rel="stylesheet"` and `<link>` is a void element that doesn't need to be closed. See https://github.com/hypothesis/h/pull/9615.